subsystem/day2_cluster_test.go

```diff
@@ -0,0 +1,326 @@
+package subsystem
+
+import (
+	//"archive/tar"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	//"io"
+	//"io/ioutil"
+	//"os"
+	//"os/exec"
+	//"reflect"
+	"time"
+
+	//"github.com/openshift/assisted-service/internal/bminventory"
+	//"github.com/openshift/assisted-service/internal/host"
+
+	//"github.com/alecthomas/units"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/google/uuid"
+
+	//"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/assisted-service/client/installer"
+	"github.com/openshift/assisted-service/models"
+)
+
+const (
+	statusInfoAddingHosts = "cluster is adding hosts to existing OCP cluster"
+)
+
+var _ = Describe("Day2 cluster tests", func() {
+	ctx := context.Background()
+	var cluster *installer.RegisterAddHostsClusterCreated
+	var clusterID strfmt.UUID
+	var err error
+	AfterEach(func() {
```

Comment:
just to keep consistency, probably worth moving it the 'before' funcs (line 65)

Reply:
fixed